__tests__/Form.test.js

```diff
@@ -208,4 +206,44 @@ describe('Errors', () => {
 
         expect(form.__http.defaults.baseURL).toBe(undefined);
     });
+
+    it('can override onSuccess and onFail methods by passing it in options', () => {
+        form = new Form({}, { onSuccess: () => 'foo', onFail: () => 'bar' });
+
+        expect(form.onSuccess()).toBe('foo');
+        expect(form.onFail()).toBe('bar');
+    });
+
+    it('can call directly HTTP verbs to submit', () => {
+        form.submit = (...args) => args;
+
+        expect(form.post('url')).toEqual(['post', 'url']);
+        expect(form.put('url')).toEqual(['put', 'url']);
+        expect(form.patch('url')).toEqual(['patch', 'url']);
+        expect(form.delete('url')).toEqual(['delete', 'url']);
+    });
+
+    it('transforms the data ta a FormData object if there is a File', async () => {
```

Comment:
@mathieutu

> transforms the data **ta** a


There is a typo in this

Reply:
Yeah, thanks!